safe/gis/utilities.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+"""Common utilities function for GIS."""
```

Comment:
Any difference between 'safe/gis/tools.py' ? Why not in this one?

Reply:
I took it from `safe/test/utilities.py`, since I wanted to move it to non-test, and the best one is `gis`, so I just naturally put it in `safe/gis/utilities` *same file name. But, i think it's better to put it in `tools.py`, less file :). I wasn't aware that tools.py is similar 

I will move it.